### DISCUSSION TITLE
Decouple the fragrance name from the fragrance selector

### DISCUSSION THREAD

**Discussion by FlyersPh9** [natekspencer/hacs-pura#30] [Ideas]
I have a "I'm home" scene that turns on the diffusers.  Right now they are set to select "Slot 1: Santorini".
![Screenshot 2023-10-06 at 11 07 35 AM](https://github.com/natekspencer/hacs-pura/assets/849817/33eabdfc-c759-4f13-8492-e0c485307bac)

But if I change the fragrance, this scene will stop working (if I understand correctly) until I update the select to use "Slot 1: New fragrance name".  What if we removed the fragrance name from the selector so scenes will use Slot 1 no matter what scent is active?

I am making some assumptions here & there may be a way to do this already.  Apologies if that is the case.

**Comment by FlyersPh9** [comment]
Any thoughts on this?  I was wondering why my Pura diffusers weren't turning on in the morning and remembered it was because I changed the scent in slot 1.  I was using Santorini but am now using a Pumpkin scent.  Because the scent changed and the automation was looking explicitly for Santorini, the automation was failing.  It would be nice if I could trigger Slot 1 despite the scent.

**Reply by natekspencer** [reply]
Thanks for the bump on this. Last month was very busy for me so I lost track of some of the things I wanted to get back to. I do agree that these should probably be decoupled so that things don't break. Initially I was trying to make it easier to see what you were selecting in Home Assistant by including the name since the UI isn't always friendly in that regard. And because I exclusively utilize the timer feature (which does support the slot number instead of a name) for my use cases, it's never been an issue.

**Reply by natekspencer** [reply]
This has been implemented in 0.1.16

**Reply by FlyersPh9** [reply]
Amazing, I will pull the update this weekend and try it out!  Thank you for the fast turn around, I really appreciate your work.

**Comment by xAcrosonicx** [comment]
Hello! Is there a repository or reference somewhere for all the fragrance names? Slot 1 currently shows "Volcano" but slot 2 shows "Fragrance code: LBA" for the Lavendar scent.

Thanks!

**Reply by natekspencer** [reply]
Please see this issue: https://github.com/natekspencer/pypura/issues/1

**Reply by xAcrosonicx** [reply]
Done - thanks so much for your help, and all the work you put into this! :)

**Reply by natekspencer** [reply]
That's available in the latest release now.

**Reply by xAcrosonicx** [reply]
Wow - thank you for the quick work! It's working perfectly now, thank you!
